emsdk.py

```diff
@@ -3023,7 +3016,7 @@ def print_tools(t):
       outfile = sys.argv[2]
     tools_to_activate = currently_active_tools()
     tools_to_activate = process_tool_list(tools_to_activate, log_errors=True)
-    env_string = construct_env(tools_to_activate, len(sys.argv) >= 3 and 'perm' in sys.argv[2])
```

Comment:
I also noticed this strange line.  I think we can maybe split this out as a separate change to simplify this one?   As far as I know this "perm" argument is not documented or used, right @juj?

Reply:
That and I removed the secondary argument for that function.

Reply:
I split that part out into separate change #525, in the hope that it will simplify this one.    If you'd rather do that yourself (to avoid re-basing on top of my change) please go ahead.